guides/css.md

```diff
@@ -0,0 +1,18 @@
+# Alternatives to `$.css()`
+
+
+## Option 1:
+
+Use getComputedStyle instead.
```

Comment:
Should note that `getcomputedstyle` triggers reflow/recalc

https://gist.github.com/paulirish/5d52fb081b3570c81e3a#getcomputedstyle